packages/vscode-extension/src/dependency/DependencyManager.ts

```diff
@@ -428,9 +425,12 @@ export class DependencyManager implements Disposable {
 export async function checkIfCLIInstalled(cmd: string, options: Record<string, unknown> = {}) {
   try {
     // We are not checking the stderr here, because some of the CLIs put the warnings there.
-    const { stdout } = await command(cmd, { encoding: "utf8", ...options });
-    return !!stdout.length;
+    const { stdout } = await command(cmd, { encoding: "utf8", quiet: true, ...options });
```

Comment:
`quiet` name needs to change eventually to something like `printStdErr`, I'd assume if quiet is true we only get status code on exit.